### DISCUSSION TITLE
エディタの検索機能で上下移動アイコンの順序に違和感がある

### DISCUSSION THREAD

**Discussion by munierujp** [increments/qiita-discussions#365] [要望・アイデア・不具合]
エディタの検索機能で上下移動アイコンが下、上の順に並んでいます。

<img width="274" alt="スクリーンショット 2022-11-24 16 54 13" src="https://user-images.githubusercontent.com/20086673/203724817-7ab30bd6-6b4f-49ed-82c7-bf6ebb54947a.png">

一般的には上、下の順であることが多いように思い、違和感があります。

Chrome:
<img width="364" alt="スクリーンショット 2022-11-24 16 56 11" src="https://user-images.githubusercontent.com/20086673/203725165-4aef0460-a9d0-4825-999a-b04e36004aa0.png">

Google スプレッドシート:
<img width="346" alt="スクリーンショット 2022-11-24 16 58 10" src="https://user-images.githubusercontent.com/20086673/203725575-8bb2e7ba-1d51-438e-9b59-8efaeee71ecf.png">

VS Code:
<img width="429" alt="スクリーンショット 2022-11-24 16 55 52" src="https://user-images.githubusercontent.com/20086673/203725095-907550e6-c15a-49fa-baf6-4e7e4e51329b.png">

**Comment by tanaka-asami415** [comment]
@munierujp 
ご意見ありがとうございます🙏
CodeMirrorというツールを使っており、元々の順番になるのですが
確かに一般的には上、下の順が多いと思いますので、どこかで変更したいと思います！🙇

**Reply by tanaka-asami415** [reply]
@munierujp 
お待たせしました🙇こちら修正が完了いたしました！
